addon/components/inputs/datetime.js

```diff
@@ -22,6 +25,15 @@ export default AbstractInput.extend({
 
   // == Computed Properties ====================================================
 
+  @readOnly
+  @computed('cellConfig.renderer.options.defaultToCurrentDateTime')
+  defaultToCurrentDateTime (defaultToCurrentDateTime) {
```

Comment:
Can't you just set the default as a defaulted parameter.

Reply:
I'm not sure exactly what you mean here, can you elaborate?

Reply:
Oh, yeah that's cleaner, I'll do that.